core/services/llo/mercurytransmitter/server.go

```diff
@@ -187,6 +187,10 @@ func (s *server) spawnTransmitLoop(stopCh services.StopChan, wg *sync.WaitGroup,
 			if t == nil {
 				// queue was closed
 				return false
+			} else if t.Report.Info.ReportFormat == llotypes.ReportFormatCapabilityTrigger {
```

Comment:
We don't need that `else`.
```suggestion
			}
			if t.Report.Info.ReportFormat == llotypes.ReportFormatCapabilityTrigger {
```